tinygrad/codegen/kernel.py

```diff
@@ -656,7 +649,8 @@ def fix_st(st: ShapeTracker, wd_pattern, tcd_pattern):
 
       return ret
 
-    return graph_rewrite(fixup_ast(self.ast), view_left)
+    return graph_rewrite(fixup_ast(self.ast),  merge_views + PatternMatcher([
```

Comment:
I would not expect a tensor core change to include this, if you really want this it should be a standalone PR.

Reply:
Did you mean the change to the schedule's view_right spec? I'll move that to a standalone PR